### ISSUE TITLE
X -axis and Y -axis of graphs obtained while training ml-agents in tensorflow

### ISSUE THREAD

**Issue by nirmalbaby1113** [Unity-Technologies/ml-agents#1728]
What are the X-axis and Y-axis for the eight graphs (Lesson,Cumulative reward,entropy,policy loss,value loss,learning rate,episode length and value estimate) which is obtained while training the ml-agents in tensorflow?

**Comment by awjuliani** [comment]
Hi @nirmalbaby1113 

Please refer to this document for details on the values shown in Tensorboard: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Using-Tensorboard.md

**Comment by awjuliani** [comment]
Due to inactivity, I am closing this issue for now. Please feel free to re-open if you deem it necessary.
